src/app/tabs/compile-tab.js

```diff
@@ -14,9 +14,9 @@ var remixLib = require('remix-lib')
 var styleGuide = remixLib.ui.themeChooser
 var styles = styleGuide.chooser()
 
-function compileTab (container, appAPI, appEvents, opts) {
-  if (typeof container === 'string') container = document.querySelector(container)
-  if (!container) throw new Error('no container given')
+function compileTab (api = {}, events = {}, opts = {}) {
+  var appAPI = api
```

Comment:
by changing the argument from `appAPI` to `api`, you have to do this ^
please keep appAPI here every occurence (if not to many) of `appAPI` to `api`

Reply:
the idea is to change every module signature to `(api = {}, events = {}, opts = {})` according to your PR to update best practices.
Personally I think it matters that the arguments are named the same in every module, so that people get used to rely on `api`, `events` and `opts` and not have different name in each module.

I see this PR as an opportunity to come closer to that goal